### ISSUE TITLE
The Play Store installed ATM of flashing but isn't visible in the drawer afterwards a reboot.

### ISSUE THREAD

**Issue by ThatAnnoyingVegan** [nift4/microg_installer_revived#117]
Hey Nick, I have some weird issues with the Play Store, the PS is visible in the app drawer until the zip is flashed and rebooted, after that it won't even install through the package installer and throws this error **"App not installed as package appears to be invalid."** but it did install before the installation of MicroG installer revived. I'll leave some screenshots they may be conducive to figuring out a fix. Thank you.

- Device: Poco F1
- OS: Android 13, Proton Plus 13.2-Stable | OFFICIAL Vanilla
- Security Patch: August 5, 2023
- Module version: 3.4.2/3.4.1/3.4.0
- Magisk: V26.4/27
- PlayStore version: https://nanolx.org/fdroid/repo/Phonesky_132.apk 👀

![Screenshot_20240207-134233_Magisk.png](https://github.com/nift4/microg_installer_revived/assets/155138818/a5014fc1-93aa-4e5c-89b1-81b71275393e)

![Screenshot_20240207-171316_Package_installer.png](https://github.com/nift4/microg_installer_revived/assets/155138818/04eac9a0-2775-4128-8cf8-d6d76a3bd020)

![Screenshot_20240207-183625_ProtonPlus_Launcher.png](https://github.com/nift4/microg_installer_revived/assets/155138818/dd01201b-55fb-4b35-b707-4a6d26ba187a)

![Screenshot_20240207-183619_ProtonPlus_Launcher.png](https://github.com/nift4/microg_installer_revived/assets/155138818/37a1dd30-18d0-463d-a19b-2f2093facd06)

![Screenshot_20240207-132222_Magisk.png](https://github.com/nift4/microg_installer_revived/assets/155138818/cc32efa1-7b1c-457e-b541-acabdf5b3c80)

![Screenshot_20240207-132521_MiXplorer.png](https://github.com/nift4/microg_installer_revived/assets/155138818/3cb6e20f-154a-482e-a887-1618f7d490db)

![Screenshot_20240207-132459_MiXplorer.png](https://github.com/nift4/microg_installer_revived/assets/155138818/f7b9049e-8ef4-4900-a601-f59ffbcc89b9)

**Comment by nift4** [comment]
Did you put Phonesky.apk into /system/priv-app or into /data/adb?

**Comment by ThatAnnoyingVegan** [comment]
> Did you put Phonesky.apk into /system/priv-app or into /data/adb?

Into /data/adb

**Comment by nift4** [comment]
Can you try to use `adb install` (on PC) / `pm install` (on Termux or similar, as root) to install the apk to observe the exact error?

**Comment by ThatAnnoyingVegan** [comment]
> Can you try to use `adb install` (on PC) / `pm install` (on Termux or similar, as root) to install the apk to observe the exact error?

Yes, I shall use the Termux, pm install "apk path" right?

**Comment by nift4** [comment]
Yes

**Comment by ThatAnnoyingVegan** [comment]
$ su
:/data/data/com.termoneplus/app_HOME # pm install /data/adb/Phonesky.apk
avc:  denied  { read } for  scontext=u:r:system_server:s0 tcontext=u:object_r:adb_data_file:s0 tclass=file permissive=0
System server has no access to read file context u:object_r:adb_data_file:s0 (from path /data/adb/Phonesky.apk, context u:r:system_server:s0)
Error: Unable to open file: /data/adb/Phonesky.apk
Consider using a file under /data/local/tmp/
Error: Can't open file: /data/adb/Phonesky.apk

Exception occurred while executing 'install':
java.lang.IllegalArgumentException: Error: Can't open file: /data/adb/Phonesky.apk
        at com.android.server.pm.PackageManagerShellCommand.setParamsSize(PackageManagerShellCommand.java:608)
        at com.android.server.pm.PackageManagerShellCommand.doRunInstall(PackageManagerShellCommand.java:1442)
        at com.android.server.pm.PackageManagerShellCommand.runInstall(PackageManagerShellCommand.java:1408)
        at com.android.server.pm.PackageManagerShellCommand.onCommand(PackageManagerShellCommand.java:221)
        at com.android.modules.utils.BasicShellCommandHandler.exec(BasicShellCommandHandler.java:97)
        at android.os.ShellCommand.exec(ShellCommand.java:38)
        at com.android.server.pm.PackageManagerService$IPackageManagerImpl.onShellCommand(PackageManagerService.java:6038)
        at android.os.Binder.shellCommand(Binder.java:1049)
        at android.os.Binder.onTransact(Binder.java:877)
        at android.content.pm.IPackageManager$Stub.onTransact(IPackageManager.java:4313)
        at com.android.server.pm.PackageManagerService$IPackageManagerImpl.onTransact(PackageManagerService.java:6022)
        at android.os.Binder.execTransactInternal(Binder.java:1285)
        at android.os.Binder.execTransact(Binder.java:1244)!


[Screenshot_20240208-204213_MiXplorer.png](https://github.com/nift4/microg_installer_revived/assets/155138818/05e9e0df-7560-495b-a40b-15b4b3382ce1)

**Comment by ThatAnnoyingVegan** [comment]
$ su
:/data/data/com.termoneplus/app_HOME # pm install /data/adb/Phonesky.apk
avc:  denied  { read } for  scontext=u:r:system_server:s0 tcontext=u:object_r:adb_data_file:s0 tclass=file permissive=0
System server has no access to read file context u:object_r:adb_data_file:s0 (from path /data/adb/Phonesky.apk, context u:r:system_server:s0)
Error: Unable to open file: /data/adb/Phonesky.apk
Consider using a file under /data/local/tmp/
Error: Can't open file: /data/adb/Phonesky.apk

Exception occurred while executing 'install':
java.lang.IllegalArgumentException: Error: Can't open file: /data/adb/Phonesky.apk
        at com.android.server.pm.PackageManagerShellCommand.setParamsSize(PackageManagerShellCommand.java:608)
        at com.android.server.pm.PackageManagerShellCommand.doRunInstall(PackageManagerShellCommand.java:1442)
        at com.android.server.pm.PackageManagerShellCommand.runInstall(PackageManagerShellCommand.java:1408)
        at com.android.server.pm.PackageManagerShellCommand.onCommand(PackageManagerShellCommand.java:221)
        at com.android.modules.utils.BasicShellCommandHandler.exec(BasicShellCommandHandler.java:97)
        at android.os.ShellCommand.exec(ShellCommand.java:38)
        at com.android.server.pm.PackageManagerService$IPackageManagerImpl.onShellCommand(PackageManagerService.java:6038)
        at android.os.Binder.shellCommand(Binder.java:1049)
        at android.os.Binder.onTransact(Binder.java:877)
        at android.content.pm.IPackageManager$Stub.onTransact(IPackageManager.java:4313)
        at com.android.server.pm.PackageManagerService$IPackageManagerImpl.onTransact(PackageManagerService.java:6022)
        at android.os.Binder.execTransactInternal(Binder.java:1285)
        at android.os.Binder.execTransact(Binder.java:1244)


![2024-02-08-21-20-09-994.log.txt](https://github.com/nift4/microg_installer_revived/files/14211966/2024-02-08-21-20-09-994.log.txt)

![Screenshot_20240208-210520_Settings.png](https://github.com/nift4/microg_installer_revived/assets/155138818/016bdd26-2820-4dfc-be76-f87f34e9ea32)

![Screenshot_20240208-211815_MiXplorer.png](https://github.com/nift4/microg_installer_revived/assets/155138818/df8f6519-cc51-4d01-b362-7b08dd5edb47)

![Screenshot_20240208-211718_MiXplorer.png](https://github.com/nift4/microg_installer_revived/assets/155138818/e46bddcc-9a75-482b-a5ef-6d9de11933ca)

![Screenshot_20240208-210553_Settings.png](https://github.com/nift4/microg_installer_revived/assets/155138818/1b9ae654-165e-4327-a20b-c6b6f9db49a3)

![Screenshot_20240208-210538_Settings.png](https://github.com/nift4/microg_installer_revived/assets/155138818/438cd9c3-7196-4b8e-a4bd-aa32a7c619f0)

![Screenshot_20240208-211357_MiXplorer.png](https://github.com/nift4/microg_installer_revived/assets/155138818/1d1907a2-706b-460f-8e83-b7bfba44da00)

**Comment by nift4** [comment]
You appear to have installed Play Store in another user or work profile, please uninstall it.

**Comment by ThatAnnoyingVegan** [comment]
> You appear to have installed Play Store in another user or work profile, please uninstall it.

Wow! I see in the guest profile the version 30.something appears but the app itself is not launchable and has this Android logo icon, with the deletion of the MicroG revived module it's gone.

**Comment by ThatAnnoyingVegan** [comment]
Just installed Minimal MicroG MinimalAP Edition updately CI build through Magisk 27. It works. PS is present and working.

**Comment by ThatAnnoyingVegan** [comment]
I think I figured it out. I don't know why it wouldn't let me use the old modded PS. Since you recommended in the Read.me to get the Play Store from APK MIRROR, but never mentioned that I'm supposed to get the NON-BUNDLE type variant, the bundle type had been causing issues because it comes with the extension .apkm which if is renamed to .apk the package gets corrupted and won't install, if it is renamed to Phonesky.apkm and put into the ```/data/adb```directory while flashing the zip won't recognize the apkm file so it proceeds with installing the companion app. That being saiou must update the Readme.
Tested back to back installing these two versions 

com.android.vending_38.8.24-29_0_PR_591260044-83882420_minAPI29(arm64-v8a,armeabi-v7a,x86,x86_64)(nodpi)_apkmirror.com.apk

com.android.vending_39.5.19-29_0_PR_602891215-83951920_minAPI29(arm64-v8a,armeabi-v7a,x86,x86_64)(nodpi)_apkmirror.com.apk
